packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -40,7 +40,7 @@ class CoverageCollector {
     });
 
     printTrace('pid $pid (port $port): collecting coverage data...');
-    final Map<dynamic, dynamic> data = await collect(host.address, port, false, false);
+    final Map<String, dynamic> data = await collect(host.address, port, false, false);
```

Comment:
this one's probably fine, assuming collect is defined to return a string->dynamic map.